src/features/workflow/modal/SplashModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { Carousel } from 'react-responsive-carousel'
+
+import Button from '../../../chrome/Button'
+import { clearModal } from '../../app/state/appActions'
+import IconButton from '../../../chrome/IconButton'
+
+const SignUpModal: React.FC = () => {
+  const dispatch = useDispatch()
+
+  const handleClose = () => {
+    dispatch(clearModal())
+  }
+
+  return (
+    <div className='bg-white p-8 text-left text-qrinavy' style={{ width: '440px'}}>
+      <div className='flex'>
+        <div className='flex-grow text-3xl font-black mb-8'>Welcome to the workflow editor!</div>
+        <IconButton icon='close' className='ml-10' onClick={handleClose}/>
+      </div>
+
+      <Carousel
+        showStatus={false}
+        showArrows={false}
+        showThumbs={false}
+        interval={3000}
+        showIndicators
+        infiniteLoop
+        autoPlay
```

Comment:
should this autoPlay? Most examples I've seen of this UX in the wild require me to scroll